### PR TITLE
Liveblogs on fronts: AB test, v2

### PR DESCRIPTION
**Pull request by stephanfowler**
- runs on all fronts
- runs at all breakpoints
- waits until card is in viewport before animating
- has a simple variant showing only an "Updated 5m ago" message
- 10% per variant

Opt-in `#ab-LiveblogBlocksOnFronts=blocks`:

liveblog-front-message-variant

Opt-in `#ab-LiveblogBlocksOnFronts=message`:

liveblog-front-b-variant png

### CODE REVIEW

**Comment by robertberry** [comment]
I like the blocks one

File: static/src/javascripts/projects/common/utils/detect.js [hunk 1]
```diff
@@ -252,15 +254,20 @@ define([
...
+            body = document.getElementsByTagName('body')[0];
+
+            viewportCache = {
+                width:  window.innerWidth  || docEl.clientWidth  || body.clientWidth,
+                height: window.innerHeight || docEl.clientHeight || body.clientHeight
+            };
+        }
+
+        return viewportCache;
     }
```
Comment (robertberry) [inline static/src/javascripts/projects/common/utils/detect.js]: Why are you caching the viewport? People can resize their browsers

Comment (stephanfowler) [inline static/src/javascripts/projects/common/utils/detect.js]: Thought it be better to minimise viewport calculations for the 99.99% users who _don't_ resize...

**Comment by robertberry** [comment]
:+1:

**Pull request merged**